lib/src/shared/status.dart

```diff
@@ -241,5 +279,55 @@ class GrpcError implements Exception {
   int get hashCode => code.hashCode ^ (message?.hashCode ?? 17);
 
   @override
-  String toString() => 'gRPC Error ($code, $message)';
+  String toString() =>
+      'gRPC Error (code: $code, codeName: $codeName, message: $message, details: $details)';
+}
+
+/// Given a status code, return the name
+String getStatusCodeValue(int code) {
```

Comment:
Does this function have to be public? If not - consider making it private.

Reply:
Fixed in https://github.com/bottlepay/grpc-dart/commit/cd197160a01b0b1ac641eccd433d5b4f55f20f9c